### PR TITLE
new endpoint to create renditions

### PR DESCRIPTION
**Pull request by vied12**

### CODE REVIEW

File: apps/picture_renditions.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
+
+import superdesk
+
+from flask import current_app as app
+from superdesk.media.renditions import generate_renditions, get_renditions_spec
+
+
+def get_file(rendition, item):
+    if item.get('fetch_endpoint'):
```
Comment (mugurrus) [inline apps/picture_renditions.py]: Some docstrings would be nice

**Comment by mugurrus** [comment]
Some behave tests for the endpoint would be nice to have, otherwise OK :+1:

**Comment by vied12** [comment]
Just added comments and tests

**Comment by petrjasek** [comment]
:+1:

**Pull request merged**